backend/enterprise/service/license.go

```diff
@@ -160,7 +160,15 @@ func (s *LicenseService) GetPlanLimitValue(name enterpriseAPI.PlanLimit) int64 {
 	if !ok {
 		return 0
 	}
-	limit := v[s.GetEffectivePlan()]
+
+	ctx := context.Background()
```

Comment:
Will be great to pass ctx around to avoid leaks.

Reply:
Done